lnbits/extension_manager.py

```diff
@@ -389,21 +400,26 @@ def download_archive(self):
 
     def extract_archive(self):
         logger.info(f"Extracting extension {self.name} ({self.installed_version}).")
-        Path("lnbits", "upgrades").mkdir(parents=True, exist_ok=True)
-        shutil.rmtree(self.ext_upgrade_dir, True)
+        Path(settings.lnbits_extensions_path, "upgrades").mkdir(
+            parents=True, exist_ok=True
+        )
+
+        tmp_dir = Path(settings.lnbits_data_folder, "unzip-temp", self.hash)
+        shutil.rmtree(tmp_dir, True)
```

Comment:
I think this one can error (for example insufficient permissions), consider `try: except:`

Reply:
in that case it is OK for the exception to be propagated (should not be hidden)